### ISSUE TITLE
NVDA will not work with the test for Microsoft A+ Network + or any .vce test engine.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#325]
**Reported by scottbb1973 on 2009-05-27 18:33**
I notice that when I try to review the test engines for my Microsoft certs I am unable to read the test engines.

Reproduction:

There is not much reproduction except that you:
1.  Open the .vce file where ever you have located it on your hard drive.
2.  click on the .vce file and yuou must have a special test engine program to read these files.
3.  No matter how you try it is not readable by NVDA.  The questions or radio buttons for the answers and probably the answers are not avaible with NVDa.

**Comment by leonardder** [comment]
This ticket is over eight years old, seems like the user is talking about specific software which. Since the lack of following comments, I don't think it has much priority. I'd suggest a close with a won't fix. We are able to undo that if it is desired.
